pkgs/games/20kly/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchurl, python2Packages }:
```

Comment:
This should be `python` instead of `python2Packages`. The packages can then be accessed with `python.pkgs`. If it is not python3 compatible, you should add `disabled = !isPy3`.

Reply:
Another one of my style preferences: I'd format the argument list like this:

```
{ stdenv
, fetchurl
, python2Packages
}:
```

Because of the same reasons as the list formatting in #45057.